core/adminserver/admin_server_test.go

```diff
@@ -255,6 +255,17 @@ func TestDelete(t *testing.T) {
 		if got, want := domain.Map.Deleted, true; got != want {
 			t.Errorf("Map.TreeState: %v, want %v", got, want)
 		}
+		// Garbage collect
```

Comment:
nit: Redundant comment, the next line is `GarbageCollect`.

Reply:
done